conf/jvm-server.options

```diff
@@ -181,8 +181,8 @@
 ###################################
 
 # Defines how to handle INSERT requests with TTL exceeding the maximum supported expiration date:
-# * REJECT: this is the default policy and will reject any requests with expiration date timestamp after 2038-01-19T03:14:06+00:00.
-# * CAP: any insert with TTL expiring after 2038-01-19T03:14:06+00:00 will expire on 2038-01-19T03:14:06+00:00 and the client will receive a warning.
+# * REJECT: this is the default policy and will reject any requests with expiration date timestamp after 2106-02-07T06:28:13+00:00 or 2038-01-19T03:14:06+00:00 depending on compatibility mode
+# * CAP: any insert with TTL expiring after 2106-02-07T06:28:13+00:00 or 2038-01-19T03:14:06+00:00 will expire on 2106-02-07T06:28:13+00:00 or 2038-01-19T03:14:06+00:00 and the client will receive a warning.
```

Comment:
We could give some additional details about the two possible max dates, and split the long lines:
```suggestion
# * REJECT: this is the default policy and will reject any requests with expiration date timestamp after
#   2106-02-07T06:28:13+00:00 or 2038-01-19T03:14:06+00:00 depending on compatibility mode (<="nc" sstable formats)
# * CAP: any insert with TTL expiring after 2106-02-07T06:28:13+00:00 or 2038-01-19T03:14:06+00:00, depending on
#   compatibility mode (<="nc" sstable formats), will expire on 2106-02-07T06:28:13+00:00 or 2038-01-19T03:14:06+00:00
    and the client will receive a warning.
```

Reply:
Done